Source/Plugins/EditorModules/ObjectInspector/Modules/ObjectInspector.cs

```diff
@@ -53,9 +54,36 @@ public bool EmptySelection
 		public ObjectInspector(int runtimeId)
 		{
 			this.InitializeComponent();
+			this.buttonLock.CheckedChanged += (sender, args) => this.UserSettings.Locked = this.buttonLock.Checked;
```

Comment:
The winforms designer is not working for me anymore so I added the changed handler myself here.

Not exactly sure why its not working anymore on visual studio 16.6.5.

Reply:
Since the designers failure is not "by design" (ha.. sorry 😅), but will likely be fixed in some future version, I'd prefer to put the event registration line manually into the proper spot inside `InitializeComponent` and make the registered method an actual named method in the class following the naming scheme. That way, the designer will pick it up without issues once it starts working again.